build

```diff
@@ -37,7 +40,7 @@ pushd "$SCRIPT_PATH" > /dev/null
 export LD_LIBRARY_PATH="${SCRIPT_PATH}/target/lib/"
 
 # build the executable with Maven
-mvn package "$@"
+LD_LIBRARY_PATH="${SCRIPT_PATH}/target/lib/" mvn package "$@"
```

Comment:
`LD_LIBRARY_PATH` is already exported as environment variable, you don't need to set it here.